plugins/email/src/main/java/de/unistuttgart/iaas/amyassist/amy/plugin/email/EMailResource.java

```diff
@@ -96,7 +96,7 @@ public String printPlainTextMessages(@PathParam("important") String important,
 			plains = this.logic.printImportantMessages(amount);
 		}
 		if (plains.equals("")) {
-			throw new WebApplicationException("Could not fetch messages from inbox.", Status.CONFLICT);
+			return "No messages found";
```

Comment:
Can you throw new WebApplicationException("No messages found.", Status.NOT_FOUND)